Add more detailed change notes

benjamin-mortimer opened the pull request:
Updated dummy change notes on mainstream prototype (National Insurance). Made to look like realistic notes a publisher might write to account for the content edits made on each edition - deliberately a bit clunky.

benjamin-mortimer committed:
Add more detailed change notes

Updated dummy change notes to look like realistic notes a publisher might write to account for the content edits made on each edition - deliberately a bit clunky.

kevindew commented:
Looks good. Do you want to also edit this one: https://github.com/kevindew/govuk-content-history-user-research/blob/404a0043dafc46b13f0f4941291f6ad31dc608f5/self-employed-national-insurance-rates/_footer.html#L29 ?

benjamin-mortimer commented:
Whoops, missed that.

benjamin-mortimer committed:
Edit more detailed change notes

Moved note from L46 to remaining dummy slot on L29 and added new note on L46. Notes now complete and reflect changes made on the right dates.

benjamin-mortimer commented:
All there now.

kevindew committed:
Merge branch 'master' into content-edits
